apps/gruenbuch/models.py

```diff
@@ -20,7 +20,8 @@ class GruenbuchOverviewPage(MetadataPageMixin, Page):
         blank=True,
         on_delete=models.SET_NULL,
         related_name='+',
-        help_text='Image ratio has to be 2:1 or 4:3.'
+        help_text='''Image ratio has to be 2:1 or 4:3. Intro image is not
```

Comment:
the triple quotes seem to insert a newline in the migration, is that on purpose?